istioctl/cmd/testdata/auth/upgrade/one-rule-multiple-services.golden.yaml

```diff
@@ -1,5 +1,12 @@
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: my-root-namespace
+spec:
```

Comment:
This may be rejected by the validation webhook, could you double check?

Maybe change to:
```
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: deny-all
  namespace: my-root-namespace
spec:
  {}
```

Reply:
Done. thanks!

Reply:
Actually, 
```
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: deny-all
  namespace: my-root-namespace
spec:
```
works too

Reply:
```
kubectl apply -f - <<EOF
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: deny-all
  namespace: istio-system
spec:
EOF

authorizationpolicy.security.istio.io/deny-all configured
```

Reply:
However, I remember something similar happened in our bug bash, right?